cluster/eval_server/launch_eval.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+sys.path.insert(0, '.')
+
+from cluster.evaluator import launch_eval
+
+import datetime
+import json
+import os
+import time
+import re
+
+import fire
+from absl import flags
+from tensorflow import gfile
+
+
+def launch_eval_job(tag, m1_path, m2_path, job_name, completions):
+    """Launches an evaluator job.
+    tag: name for later reference
+    m1_path, m2_path: full gs:// paths to the .pb files to match up
+    job_name: string, appended to the container, used to differentiate the job
+    names (e.g. 'minigo-cc-evaluator-v5-123-v7-456')
+    completions: the number of completions desired (each completion is 2 games)
+    """
+    print()
+    if not re.match(r'[a-z0-9-]*$', tag, re.I):
+        print("{} is not a valid tag".format(tag))
+        return
+
+    # TODO: Change to minigo-pub
+    bucket_path = "gs://sethtroisi-sandbox/experiments/eval/" + tag
+
+    ######## WRITE DOWN TO BUCKET ########
+
+    metadata_path = os.path.join(bucket_path, 'metadata')
+    assert not gfile.Exists(metadata_path), "Already exists"
+
+    TS=str(int(time.time()))
+    with gfile.GFile(os.path.join(bucket_path, 'commands_' + TS), "w") as commands:
+        commands.write(str(sys.argv) + "\n")
+
+    metadata = {
```

Comment:
you should probably also grab some of the job spec from the launch_eval job that is returned:  among other things, it would include the name & id of the image run, which would be needed.  If `launch_eval.launch_eval_job` doesn't return that spec, it could be refactored to do so.